[2024-11-19T09:34:09.478+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-19T09:34:09.488+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bernadya.save_youtube_data manual__2024-11-19T09:33:17.160746+00:00 [queued]>
[2024-11-19T09:34:09.495+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bernadya.save_youtube_data manual__2024-11-19T09:33:17.160746+00:00 [queued]>
[2024-11-19T09:34:09.495+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-19T09:34:09.511+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): save_youtube_data> on 2024-11-19 09:33:17.160746+00:00
[2024-11-19T09:34:09.515+0000] {standard_task_runner.py:72} INFO - Started process 195 to run task
[2024-11-19T09:34:09.518+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'bernadya', 'save_youtube_data', 'manual__2024-11-19T09:33:17.160746+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmptrao7jpq']
[2024-11-19T09:34:09.520+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask save_youtube_data
[2024-11-19T09:34:09.532+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T09:34:09.557+0000] {task_command.py:467} INFO - Running <TaskInstance: bernadya.save_youtube_data manual__2024-11-19T09:33:17.160746+00:00 [running]> on host 368b98dccfcf
[2024-11-19T09:34:09.623+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bernadya' AIRFLOW_CTX_TASK_ID='save_youtube_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-19T09:33:17.160746+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-19T09:33:17.160746+00:00'
[2024-11-19T09:34:09.624+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-19T09:34:09.664+0000] {logging_mixin.py:190} INFO - Successfully connected to MongoDB.
[2024-11-19T09:34:09.668+0000] {logging_mixin.py:190} INFO - Existing data found, moving to historical collection...
[2024-11-19T09:34:09.670+0000] {logging_mixin.py:190} INFO - Existing data moved to historical collection.
[2024-11-19T09:34:09.671+0000] {logging_mixin.py:190} INFO - New data for channel statistics successfully saved to MongoDB.
[2024-11-19T09:34:09.672+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-19T09:34:09.678+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-19T09:34:09.679+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=bernadya, task_id=save_youtube_data, run_id=manual__2024-11-19T09:33:17.160746+00:00, execution_date=20241119T093317, start_date=20241119T093409, end_date=20241119T093409
[2024-11-19T09:34:09.731+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-19T09:34:09.741+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-19T09:34:09.742+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
